.github/workflows/docker-image-build-push.yml

```diff
@@ -24,8 +24,13 @@ jobs:
         with:
             fetch-depth: 0
 
+      - name: Set up QEMU
```

Comment:
Curious what this is for and how it relates to the snapshots?

Reply:
just accidently pushed changes from different branch - removed